source/game_sa/CRadar.h

```diff
@@ -142,20 +141,26 @@ VALIDATE_SIZE(airstrip_info, 0x10);
 
 
 struct tRadarTrace {
-    unsigned int   m_dwColour; // see eBlipColour
-    unsigned int   m_dwEntityHandle;
+    eBlipColour    m_nColour;
+    unsigned int   m_nEntityHandle;
     CVector        m_vPosition;
     unsigned short m_nCounter;
     float          m_fSphereRadius;
     unsigned short m_nBlipSize;
     class CEntryExit* m_pEntryExit;
     unsigned char  m_nBlipSprite; // see eRadarSprite
-    unsigned char  m_bBright : 1; // It makes use of bright colors. Always set.
-    unsigned char  m_bTrackingBlip : 1; // It is available.
-    unsigned char  m_bShortRange : 1; // It doesn't show permanently on the radar.
-    unsigned char  m_bFriendly : 1; // It is affected by BLIP_COLOUR_THREAT.   
-    unsigned char  m_bBlipRemain : 1; // It has the priority over the entity (it will still appear after the entity's deletion).
-    unsigned char  m_bBlipFade : 1; // Possibly a leftover. Always unset (unused).
+    union {
+        struct {
+            unsigned char  m_bBright : 1; // It makes use of bright colors. Always set.
+            unsigned char  m_bTrackingBlip : 1; // It is available.
+            unsigned char  m_bShortRange : 1; // It doesn't show permanently on the radar.
+            unsigned char  m_bFriendly : 1; // It is affected by BLIP_COLOUR_THREAT.
+            unsigned char  m_bBlipRemain : 1; // It has the priority over the entity (it will still appear after the entity's deletion).
+            unsigned char  m_bBlipFade : 1; // Possibly a leftover. Always unset (unused).
+            unsigned char  pad : 2;
+        };
+        unsigned char m_bCreationFlags;
+    };
```

Comment:
```suggestion
        };
        unsigned char m_bCreationFlags;
    };
```
You don't need  `unsigned char m_bCreationFlags` here. Just use the bit fields.